src/main/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -364,10 +364,15 @@ public int scrub(boolean disableSnapshot, boolean skipCorrupted, boolean checkDa
     public int scrub(boolean disableSnapshot, boolean skipCorrupted, boolean checkData, int jobs, String keyspaceName,
             String... columnFamilies) throws IOException, ExecutionException, InterruptedException;
 
+    @Deprecated
     public int scrub(boolean disableSnapshot, boolean skipCorrupted, boolean checkData, boolean reinsertOverflowedTTL,
             int jobs, String keyspaceName, String... columnFamilies)
             throws IOException, ExecutionException, InterruptedException;
 
+    public int scrub(boolean disableSnapshot, boolean skipCorrupted, String scrubMode, boolean checkData, boolean reinsertOverflowedTTL,
+            int jobs, String keyspaceName, String... columnFamilies)
+            throws IOException, ExecutionException, InterruptedException;
```

Comment:
I think there should be a variant with `skipCorrupted` and one with `scrubMode`, doesn't really make sense to set both, as they conflict (and I don't recall how the conflict is resolved in the API).

Reply:
Hmm, that makes sense.  Our documentation says that scrubMode trumps skipCorrupted.
https://github.com/scylladb/scylla/blob/f98cb965067b8a8f640840387dcbf043a0885d8c/api/api-doc/storage_service.json#L798

Reply:
Fixed in v2